AirCasting/SDSync/MobileSessions/SDCardMobileSessionsSavingService.swift

```diff
@@ -207,6 +212,42 @@ class SDCardMobileSessionsSavingService: SDCardMobileSessionssSaver {
         }
     }
     
+    private func addMeasurementLocation(deviceID: String, measurements: SDCardMeasurementsRow) -> CLLocationCoordinate2D? {
+        // HERE - wait for Monika's geolocalization branch, then update this one and proced
```

Comment:
I don't know if you left it for a reason? 
Bc If you rebase this to geolocalization branch you can remove this, it's just waiting for Marta's approve :)